### DISCUSSION TITLE
LFS with github pages

### DISCUSSION THREAD

**Discussion by ghost** [community/community#62105] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Please help to deploy github pages with LFS videos. I tried to use tag LFS: true in Checkout in .yml file of gh workflow but LFS videos still not working.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @thaitdth, we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
